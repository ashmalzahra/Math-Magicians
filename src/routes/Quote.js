import './styles/Quote.css';

const Quote = () => (
  <div className="contain-quote">
    <p style={{ fontSize: 'xxx-large', margin: '0' }}>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. -William Paul Thurston
    </p>
  </div>
);

export default Quote;
