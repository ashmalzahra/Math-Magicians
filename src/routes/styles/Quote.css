.contain-quote {
  display: flex;
  justify-content: center;
  margin: 0 7%;
  top: 50%;
  position: absolute;
}
