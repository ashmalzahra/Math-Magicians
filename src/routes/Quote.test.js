import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from './Quote';
import '@testing-library/jest-dom';

describe('Checking page deployment', () => {
  test('Quote component matches snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders Quote component', () => {
    render(<Quote />);

    expect(screen.getByText(/Mathematics is not about numbers,/)).toBeInTheDocument();
  });
});
