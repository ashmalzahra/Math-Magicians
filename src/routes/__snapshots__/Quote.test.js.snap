// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking page deployment Quote component matches snapshot 1`] = `
<div
  className="contain-quote"
>
  <p
    style={
      Object {
        "fontSize": "xxx-large",
        "margin": "0",
      }
    }
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
